lib/rules/no-array-constructor.js

```diff
@@ -18,7 +18,11 @@ module.exports = {
             url: "https://eslint.org/docs/rules/no-array-constructor"
         },
 
-        schema: []
+        schema: [],
+
+        messages: {
+            preferLiteral: "The array literal notation [] is preferrable."
```

Comment:
This is inherited, but the correct spelling is "preferable" if you feel like changing it. (But not a blocker for sure)

Reply:
Fixed!